content/work/expression-builder.md

```diff
@@ -3,9 +3,10 @@ title: Expression Builder
 sub: Tools for data analytics
 logo: aunalytics
 client: &client Aunalytics
-date: 2019-05-14
+date: 2018-05-14
+end: 2019-05-14
 image:
-  src: projects/aunalytics/exbldr.jpg
+  src: work/expression-builder/exbldr.jpg
```

Comment:
@SondraE I noticed we have a few images like this (I think exbldr2.jpg and 3.jpg) that I didn't see us using. Can you confirm, and if we aren't we can probably delete those from the project.

Reply:
Also, along these lines, I think we aren't using any of these:

src/images/work/quarqnet/mockup2.png
src/images/work/quarqnet/implementation1.jpg
src/images/work/quarqnet/implementation2.jpg
src/images/work/quarqnet/qasper-splash.jpg
src/images/work/quarqnet/quarqnet-sketch-mobile.jpg
src/images/work/quarqnet/tabs.jpg
src/images/work/quarqrace/trig-mobile.jpg
src/images/work/quarqrace/trig-splash.jpg


I didn't check all of the images, and I assume we aren't using some coachhub and maybe adobe images.